packages/core/src/Page/PageVueServerRenderer.ts

```diff
@@ -1,15 +1,25 @@
-const Vue = require('vue');
-const path = require('path');
-const fs = require('fs-extra');
-
-const VueCompiler = require('vue-template-compiler');
-const { renderToString } = require('vue-server-renderer').createRenderer();
+/* eslint-disable import/no-import-module-exports */
```

Comment:
Is this due to the false positives from the `requireFromString` method? Let's document this in a comment.

Reply:
Sure, done!